.editorconfig

```diff
@@ -0,0 +1,18 @@
+root = true
```

Comment:
There probably needs to be some discussion on tge dev@ list whether or not we want to start using this. My own first impressions are positive.

Reply:
More information can be found here: https://editorconfig.org/